www/novj35/firma.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="cs">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Apple</title>
+</head>
+
+<body>
+    <header>
+        <span>Apple</span>
+        <nav>
+            <li><a href="">Domů</a></li>
+            <li><a href="">Výrobky</a></li>
+            <li><a href="">Doprava a platba</a></li>
+            <li><a href="">O nás</a></li>
+        </nav>
+    </header>
+    <h1>Apple - Does Less, Costs More</h1>
+    <ul>
```

Comment:
Tady je špatné použití tagu &lt;ul&gt;, protože pod ním musí být &lt;li&gt; a pak teprve mohou být jiné tagy

Tahle chyba se také objeví při validaci na https://validator.w3.org

Takže prosím zkus to zvalidovat a opravit